### ISSUE TITLE
add timer indicator on how long you could record

### ISSUE THREAD

**Issue by as3mbus** [phw/peek#195]
would love to see a information on how many time left i can record my screen with the available memory. it would indicate when i should start wrap things up or maybe plan what i can do with the remaining time.

Sorry for my bad english 😢

**Comment by phw** [comment]
Not sure about this one, especially as this is hard to predict. I am currently changing the GIF generation and I hope to get rid of all the resource issues Peek currently has, so this might not be necessary anymore.

Could you try the latest development version and see if this works better for you in general?

**Comment by as3mbus** [comment]
tried the source build from the master branch. still troubles me on when i should finish my recording. maybe a workaround with simple information about ram usage and it's limit that would really help.

**Comment by phw** [comment]
I have finished my refactoring of the recording. I got rid of the imagemagick post processing, which had this exorbitant memory usage. This also allowed me to refactor the recording itself and use a compressed temporary video file again.

That means both disk and RAM usage are now significantly lowered. I still don't see an easy way to have get a proper estimate of the available recording time across all supported formats and recorders, and I have no intentions for monitoring disk and memory usage, there are dedicated tools for this. So I will close this issue.

I might add a display of elapsed recording time somewhere, as I think this is useful and actually in the scope of Peek. But that's a different story :)
